packages/ui/components/index-page/MainSection.tsx

```diff
@@ -0,0 +1,13 @@
+import { Layout } from 'antd';
+import { ChildrenProps } from 'types';
+
+const MainSection = ({ children }: ChildrenProps) => {
+  const { Content } = Layout;
+
+  return (
+    <Content className="mainAreaContainer">
```

Comment:
kebab-case for CSS classes?

Reply:
@brettstack ya! sure 👍  will also follow the `BEM` style of css writing, missed this somehow.